coalib/misc/Shell.py

```diff
@@ -99,11 +99,11 @@ def get_shell_type():  # pragma: no cover
              Powershell is detected, "cmd" if command prompt is been
              detected or "sh" if it's neither of these.
     """
-    out_hostname, _ = run_shell_command(["echo", "$host.name"], shell=True)
-    if out_hostname.strip() == "ConsoleHost":
+    out = run_shell_command("echo $host.name", shell=True)[0]
```

Comment:
the change from the arg list to a string because of `shell=True` is due to another commit in this PR right? Shouldn't that one happen there?